html5lib/_utils.py

```diff
@@ -65,6 +70,41 @@ def __getitem__(self, key):
         return dict.get(self, key, self.default)
 
 
+class BoundMethodDispatcher(Mapping):
+    def __init__(self, instance, dispatcher):
```

Comment:
I think this class could use some documentation about what's going on. It took a while to page in enough to figure out.